portugues/colabora/semdiferencas/pentefino/pentefino.md

```diff
@@ -2,4 +2,4 @@ Depois de passar horas analisando e depurando, você
 
 [Descobre um erro de lógica](../../comdiferencas/diff/diff.md)
 
-[Já não consegue raciocinar e fica babando no teclado de tanto sono](../../colaborar.md)
```

Comment:
Actually, new lines at the end are the one case Casey can handle properly when auto-merging (although it used to be that she couldn't). I'm happy to merge things that look like modifications but really aren't (e.g. adding to a line). But in this case, it looks like you're updating what happens if you choose an option? I'd prefer if you created a new option to take the reader to your portion of the story, rather than changing what happens as a result of someone else's option.